src/transformers/generation/configuration_utils.py

```diff
@@ -421,6 +429,8 @@ def __init__(self, **kwargs):
         self.top_k = kwargs.pop("top_k", 50)
         self.top_p = kwargs.pop("top_p", 1.0)
         self.min_p = kwargs.pop("min_p", None)
+        self.moment_p_exponent = kwargs.pop("moment_p_exponent", 2.0)
+        self.moment_p_alpha = kwargs.pop("moment_p_alpha", 1.0)
```

Comment:
New arguments should default to `None`, so that we can distinguish e.g. a user-defined `moment_p_exponent=2.0` from the default value.

It seems equivalent now, but non-None defaults often bring issues in the log run